examples/control-elements/media-dialog.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width" />
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css" />
+    <style>
+      /** add styles to prevent CLS (Cumulative Layout Shift) */
+      media-controller:not([audio]) {
+        display: block;         /* expands the container if preload=none */
+        max-width: 960px;       /* allows the container to shrink if small */
+        aspect-ratio: 2.4;   /* set container aspect ratio if preload=none */
+      }
+
+      video {
+        width: 100%;      /* prevents video to expand beyond its container */
+      }
+
+      .examples {
+        margin-top: 20px;
+      }
+    </style>
+
+    <script type="module" src="../../dist/index.js"></script>
+    <script type="module" src="../../dist/experimental/media-dialog.js"></script>
+  </head>
+  <body>
+    <h1>Media Dialog</h1>
+    <br>
+
+    <media-controller>
+      <video
+        slot="media"
+        src="https://stream.mux.com/O4h5z00885HEucNNa1rV02wZapcGp01FXXoJd35AHmGX7g/low.mp4"
+        muted
+        crossorigin
+      ></video>
+
+      <media-control-bar>
+        <media-play-button></media-play-button>
+        <media-time-display show-duration></media-time-display>
+        <media-time-range></media-time-range>
+        <media-playback-rate-button></media-playback-rate-button>
+        <media-mute-button></media-mute-button>
+        <media-volume-range></media-volume-range>
+      </media-control-bar>
+
+      <media-dialog open>
+        <style>
+          media-dialog:not(:defined) {
+            display: none;
+          }
+          .dialog-close {
+            background: none;
+            color: inherit;
+            border: none;
+            padding: 0;
+            font: inherit;
+            cursor: pointer;
+            outline: inherit;
+            width: 28px;
+            height: 28px;
+            position: absolute;
+            top: 1rem;
+            right: 1rem;
+          }
+          .dialog-title {
+            color: #fff;
+          }
+        </style>
+
+        <button class="dialog-close" tabindex="0" onclick="this.parentNode.close()">
+          <svg fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+          </svg>
+        </button>
```

Comment:
for now I just put this in the light DOM

Reply:
![](https://i.imgflip.com/7cxpvw.jpg)

Reply:
I do find the irony amusing of using a "dark side" character for the avoiding the "shadow dom"